src/test/java/uk/gov/hmcts/reform/et/syaapi/service/CaseServiceTest.java

```diff
@@ -299,4 +303,69 @@ void shouldSubmitUpdateCaseInCcd() {
 
         assertEquals(caseDetails, expectedDetails);
     }
+
+    @Test
+    void shouldReturnCaseData() throws IOException {
```

Comment:
I would name tests as a **_when-then_** scenario.  I.e. whenGoodRequestThenReturnCaseData
If it wasn't for PMD or checkstyle, I'd even go as far as seperating the when-then with an underscore, but that's not liked here!

Reply:
Noted - I did try underscores but like you said PMD doesn't like it